static/sass/_pattern_global-nav.scss

```diff
@@ -95,7 +95,7 @@
     padding-bottom: 0.5rem;
   }
 
-  .data-fabric-menu {
+  .microsite-menu {
```

Comment:
@mtruj013 Careful about this, we have two navigations at the moment:

- MicroCloud would be a _microsite_ navigation, with a single menu bar (no narrow bar on top linking back to "Canonical". This is for migrated microsites.
- Data Fabric would be a _bubble_ navigation, with the double menu bar similar to what we have on bubble in the [meganav demo](https://ubuntu-com-12939.demos.haus/). This is for migrated bubbles from Ubuntu.com.

Mockups for both projects reflect this.

(This is the interim navigation for canonical.com)

Reply:
I made this change before I was told that I was supposed to be building a different version of the subnav, I've changed it back